sdk/eventhub/event-hubs/src/util/connectionStringUtils.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { parseConnectionString } from "@azure/core-amqp";
+
+/**
+ * The set of properties that comprise an Event Hub connection string.
+ */
+export interface EventHubConnectionStringProperties {
+  /**
+   * The fully qualified Event Hub namespace extracted from the "Endpoint" in the
+   * connection string. This is likely to be similar to "{yournamespace}.servicebus.windows.net".
+   * This is typically used to construct the EventHub{Producer|Consumer}Client.
+   */
+  fullyQualifiedNamespace: string;
+  /**
+   * The value for "Endpoint" in the connection string.
+   */
+  endpoint: string;
+  /**
+   * The value for "EntityPath" in the connection string.
```

Comment:
Copying over from SB

```suggestion
   * The value for "EntityPath" in the connection string which would be the name of the event hub instance associated with the connection string.
```